components/picker/stories/picker.stories.js

```diff
@@ -52,6 +52,15 @@ export default {
 			control: "boolean",
 			if: { arg: "labelPosition", eq: "side" },
 		},
+		showWorkflowIcon: {
```

Comment:
This addition can be found in the docs-to-storybook PR for picker: [Picker docs to storybook migration](https://github.com/adobe/spectrum-css/pull/3200)